### ISSUE TITLE
Test-extended.system-JDK10-linux_x86-64_cmprssptrs SharedClasses.SCM01.MultiThread_0 assertion CallSnippet.cpp:143: j2iThunk != NULL

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#3050]
https://ci.eclipse.org/openj9/job/Test-extended.system-JDK10-linux_x86-64_cmprssptrs/48

```
MT5 stderr Assertion failed at /home/jenkins/workspace/Build-JDK10-linux_x86-64_cmprssptrs/build/linux-x86_64-normal-server-release/vm/compiler/../compiler/x/codegen/CallSnippet.cpp:143: j2iThunk != NULL
MT5 stderr VMState: 0x0005ff0a
MT5 stderr 	null virtual j2i thunk
MT5 stderr compiling java/util/Objects.equals(Ljava/lang/Object;Ljava/lang/Object;)Z at level: cold
```

**Comment by irinarada** [comment]
@andrewcraik - can we look at this?

**Comment by irinarada** [comment]
Thanks @liqunl for looking at this!

**Comment by andrewcraik** [comment]
FYI @jdmpapin

**Comment by liqunl** [comment]
@pshipton Where can I download the test?

**Comment by liqunl** [comment]
Found the test in https://github.com/eclipse/openj9-systemtest/tree/master/openj9.test.sharedClasses

**Comment by pshipton** [comment]
Given the failure rate, it seems this can be deferred to the next release.

**Comment by pshipton** [comment]
Another occurrence https://ci.eclipse.org/openj9/job/Test-extended.system-JDK10-linux_x86-64_cmprssptrs/53

**Comment by pshipton** [comment]
Fixed by #3135
